docs/cloud/cloud-org/cloud-org-upgrade-to-paid.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrade to paid
+layout: default
+parent: Manage organization
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I add payment information for my organization?
+
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Database usage pricing](https://www.featurebase.com/pricing){:target="_blank:}
+* [Available database sizes](https://docs.featurebase.com/docs/cloud/cloud-databases/cloud-db-shape/)
+
+## Important payment information
+
+{% include /cloud-org/cloud-payment-details.md %}
+
+## Add payment information for my organization
+
+* Click **Configuration** > **Upgrade to Paid**.
+* Choose your payment method: **Card** or **US bank account**
+* Enter your credit card information or your bank account details.
```

Comment:
This isn't as simple as this as I've discovered and documented in Jira 222

https://molecula.atlassian.net/browse/DOCS-222

In short, there are 4x payment methods:

* Pay by ccard (I'm assuming this has to be a US ccard because that's not clear on the form)
* Search for a bank account
* Click a bank account button > pops up a Stripe payment form
* Click "enter details manually" link > pops up a Stripe payment form

Then there's the big blue "SUBMIT" button which may apply to any or all the above.

Reply:
Ya people are pretty savy and each bank account is different, so we're not going to document these flows, especially since we don't know them